srcalc/src/main/webapp/WEB-INF/views/tiles/enterVariables.jsp

```diff
@@ -68,12 +68,18 @@
                 <thead><tr><th>Select</th><th>CPT Code</th><th>Description</th><th>RVU</th></tr></thead>
                 <!-- We're using datatables to provide paging now, but still
                 truncate the list until we improve browser performance. -->
-                <c:forEach var="procedure" items="${srcalc:truncateList(variable.procedures, 500)}">
-                <tr>
-                <td class="selectRadio">
-                <form:radiobutton path="${varPath}" value="${procedure.cptCode}" data-display-string="${procedure}"/></td>
-                <td>${procedure.cptCode}</td><td>${procedure.longDescription}</td><td>${procedure.rvu}</td></tr>
-                </c:forEach>
+                
+                <script>
+					procedureArray = new Array();
+					<c:forEach var="procedure" items="${variable.procedures}">
+						var entry = new Array();
+						entry.push("<input type=\"radio\" name=\"${varPath}\" value=\"${procedure.cptCode}\" alt=\"${procedure}\"/>");
+						entry.push("${procedure.cptCode}");
+						entry.push("${fn:escapeXml(procedure.longDescription)}");
+						entry.push("${procedure.rvu}");
+						procedureArray.push(entry);
```

Comment:
How about just using JS literal array syntax instead of dynamically growing the array like this?